src/ui/component/splash/internal/load-screen.jsx

```diff
@@ -4,6 +4,7 @@ import React, { Fragment } from 'react';
 import Icon from 'component/common/icon';
```

Comment:
I think this entire file can be moved into `component/splash/view.jsx`.

All it really needs to be is:
```
      <div className="load-screen">
        <h1 className="load-screen__title">
          {__('LBRY')}
          <sup>beta</sup>
        </h1>
        <css-doodle use="var(--rule)" />
      </div>
```